joomla/joomla-cms
# [3.9] Add privacy note to the new captcha plugin

### Summary of Changes

Add privacy note to the new captcha plugin

### Testing Instructions

- apply this patch
- notice the privacy notice in the backend

### Expected result

There is also a privacy notice for this plugin the backend

### Actual result

There is NO privacy notice for this plugin the backend

### Documentation Changes Required

None
